examples/ioredis/tracer.js

```diff
@@ -1,17 +1,17 @@
 'use strict';
 
 const opentelemetry = require('@opentelemetry/core');
-const { NodeTracerRegistry } = require('@opentelemetry/node');
+const { NodeTracerProvider } = require('@opentelemetry/node');
 const { SimpleSpanProcessor } = require('@opentelemetry/tracing');
 const { JaegerExporter } = require('@opentelemetry/exporter-jaeger');
 
-const tracerRegistry = new NodeTracerRegistry();
+const tracerProvider = new NodeTracerProvider();
```

Comment:
I think we should be consistent everywhere, some places we used `provider` and other `tracerProvider`.

Reply:
Made it `provider` everywhere